### PR TITLE
Add some iothread virsh commands support

### PR DESCRIPTION
**Pull request by rbian**
The new commands are for Get/Add/Delete domain IOThreads, and
control domain IOThread affinity.

Signed-off-by: Ruifeng Bian <rbian@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -2855,3 +2855,35 @@ def get_vol_list(pool_name, vol_check=True, timeout=5):
...
+        3 0-7
+
+    The function return a dict like:
+
+    ::
+        {'2': '3', '1': '0-4', '4': '0-7', '3': '0-7'}
+    """
+    ret = virsh.iothreadinfo(vm_name, options, ignore_status=True)
+    if ret.exit_status:
+        raise exceptions.TestFail(ret.stderr.strip())
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: As the 'ignore_status' is True, maybe you don't need raise exception here(at least it's not a TestFail exception), and just return {}

File: virttest/virsh.py [hunk 2]
```diff
@@ -4194,3 +4194,71 @@ def click_button(name, left_button=True, **dargs):
...
+    cmd = "iothreadadd %s %s" % (name, thread_id)
+    if options:
+        cmd += " %s" % options
+
+    return command(cmd, **dargs)
+
+
+def iothreaddel(name, thread_id, options=None, **dargs):
+    """
+    Delete an IOThread to the guest domain.
```
Comment (will-Do) [inline virttest/virsh.py]: s/to/from

File: virttest/virsh.py [hunk 3]
```diff
@@ -4194,3 +4194,71 @@ def click_button(name, left_button=True, **dargs):
...
+    cmd = "iothreadinfo %s" % name
+    if options:
+        cmd += " %s" % options
+
+    return command(cmd, **dargs)
+
+
+def iothreadpin(name, thread_id, cpuset, options=None, **dargs):
+    """
+    Delete an IOThread to the guest domain.
```
Comment (will-Do) [inline virttest/virsh.py]: You forgot this one

**Comment by rbian** [comment]
Thanks @will-Do , updated the code.

**Comment by will-Do** [comment]
ACK

**Pull request merged**